code/modules/mob/living/carbon/carbon.dm

```diff
@@ -643,7 +643,7 @@
 	. = ..()
 	if(!istype(loc, /turf/space))
 		for(var/obj/item/I in get_all_slots())
-			if(I.flags & SLOWDOWN_WHEN_CARRIED)
+			if(I.slot_flags & SLOT_BACK)
```

Comment:
I don't think what was here before was correct, but neither is this. You're checking if it *can* be worn on the back, not if it *is* on the back.

Reply:
Good point, will fix